src/frontend/apps/e2e/__tests__/app-impress/utils-common.ts

```diff
@@ -83,6 +83,33 @@ export const randomName = (name: string, browserName: string, length: number) =>
     return `${browserName}-${Math.floor(Math.random() * 10000)}-${index}-${name}`;
   });
 
+const HEADER_MENU_TOGGLE_TEST_ID = 'header-menu-toggle';
+
+const getHeaderMenuButton = (page: Page) =>
+  page.getByTestId(HEADER_MENU_TOGGLE_TEST_ID);
+
+export const openHeaderMenu = async (page: Page) => {
+  const toggleButton = getHeaderMenuButton(page);
+  await expect(toggleButton).toBeVisible();
+
+  const isExpanded =
+    (await toggleButton.getAttribute('aria-expanded')) === 'true';
+  if (!isExpanded) {
+    await toggleButton.click();
+  }
+};
+
+export const closeHeaderMenu = async (page: Page) => {
+  const toggleButton = getHeaderMenuButton(page);
+  await expect(toggleButton).toBeVisible();
+
+  const isExpanded =
+    (await toggleButton.getAttribute('aria-expanded')) === 'true';
+  if (isExpanded) {
+    await toggleButton.click();
+  }
+};
```

Comment:
Let's keep it simple, wdyt about that ?

```suggestion
export const toggleHeaderMenu = async (page: Page) => {
  const toggleButton = getHeaderMenuButton(page);
  await expect(toggleButton).toBeVisible();

  await toggleButton.click();
};
```